alignak/objects/contact.py

```diff
@@ -433,12 +427,41 @@ def linkify(self, commands, notificationways):
         :return: None
         TODO: Clean this function
         """
+        self.linkify_contact_by_contactgroup(contactgroups)
         self.linkify_with_notificationways(notificationways)
         self.linkify_command_list_with_commands(commands, 'service_notification_commands')
         self.linkify_command_list_with_commands(commands, 'host_notification_commands')
 
+    def linkify_contact_by_contactgroup(self, contactgroups):
+        """Link contacts with contactgroups
+
+        :param contactgroups: contactgroups
+        :type contactgroups: alignak.objects.contactgroup.contactgroups
+        :return: None
+        """
+        for contact in self:
+            new_contactgroups = []
+            if hasattr(contact, 'contactgroups') and contact.contactgroups:
+                # Because contacts groups can be created because a contact references
+                # a not existing group or because a group exists in the configuration,
+                # we can have an uuid or a name...
+                for contactgroup_name in contact.contactgroups:
+                    if contactgroup_name in contactgroups:
```

Comment:
Yes. Two different creation modes for the groups: from the configuration or from implicit declaration in an object ... this will create the group